core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -337,11 +344,34 @@ class SocketServerTest {
       externalReadyFuture.complete(null)
       TestUtils.waitUntilTrue(() => listenerStarted(externalListener), "External listener not started")
     } finally {
+      externalReadyFuture.complete(null)
```

Comment:
Why? If it for the failure case, then perhaps it should be in a catch block?

Reply:
This is not needed. It is a left over from my debugging. Let me remove it.